MovieMVVM/MovieMVVM/Assembly/AssemblyModule.swift

```diff
@@ -16,15 +16,23 @@ final class AssemblyModule: AssemblyModuleProtocol {
     func createMainModule() -> UITableViewController {
         let view = MainTableViewController()
         let movieAPIService = MovieAPIService()
-        let viewModel = MovieViewModel(movieAPIService: movieAPIService)
+        let realmProvider = RealmProvider()
+        let repositoryProtocol = RealmRepository(realmProvider: realmProvider)
```

Comment:
```suggestion
        let repository = RealmRepository(realmProvider: realmProvider)
```
это точно у Сереги катнул у него таже ошибка